src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * File: drc_asin.h
```

Comment:
Should we follow the rule to place header files to src/include/sof/audio?

Reply:
Yes, we should / must follow . Since it is a testbench code & deattached one, i didn't kept under/follow src/include/sof/audio